docs/source/Plugin/P075.rst

```diff
@@ -276,7 +277,7 @@ It also sends new touch event values to the controller.
 :raw-html:`<br clear="all">`
 
 
-Nextion Command Format Summary
+Nextion Data Format Summary
 ------------------------------

```

Comment:
Make sure the line (to signal the (sub)section level) has the same number of characters as the title itself.

Reply:
I'll need a bit more hand holding on this since your comment is unclear.

Do you mean to change this:
```
Nextion Data Format Summary
------------------------------
```
To this:
```
Nextion Data Format Summary
---------------------------
```

If that is the issue then the existing text is suppose to be OK. The heading rules seem to confirm the extra line length is OK:
```
If under and overline are used, their length must be identical
The length of the underline must be at least as long as the title itself
```
More details here:
https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html

Reply:
OK, didn't know it could also be longer.
I noticed in the docs that Sphinx gave warnings when the underline was shorter.

Learned something myself today :)

Reply:
> Learned something myself today 

I'm new to reStructured Text, so avoid using me as a trusted teacher. :)

I like rst's pretty output. But getting it is like using the word processor on an old PDP-11 mainframe while typing on a VT100 terminal.

Reply:
Well programming the ESP isn't that different.
Amount of memory is in the same order of magnitude and debugging is still very basic.
Only main improvement is that when you let your code slip out of your hand you don't have to get on your knees to collect all punch cards again and re-order them.

I've started programming a MSX-1 from 1983, when I was 7 and back then I had more free memory compared to what we now have on the ESP8266.
But instead of mounting Lego, paperclips and aluminum foil covered pins connected to the joystick ports to count pulses, we now use slightly more fancy switches to count and the connectors are less clunky :)
So there has been some progress in 37 years I've been working with computers, electronics and programming. 👍